app/locale/et_EE/Sendsmaily_Sync.csv

```diff
@@ -1,15 +1,27 @@
 "Thank you for subscribing to our newsletter!","Aitäh, et liitusite meie uudiskirjaga!"
 "Thank you for your subscription","Aitäh, et liitusite meie uudiskirjaga!"
-"Data must be posted with POST method.","Viga andmete postitamisel."
-"Invalid or deprecated API key.","Aegunud API võti."
 "No data recieved.","Vajalikud andmed puuduvad."
 "Posted fields do not contain a valid email address.","Vigane e-posti aadress!"
-"No autoresponder data set.","Automaatvastaja puudub või on valesti seadistatud."
 "Could not deliver email.","e-kirja saatmine ebaõnnestus!"
 "Newsletter subscriptions synchronized.","Uudiskirja kontaktid sünkroniseeritud."
 "Select fields you wish to synchronize along with subscription data","Vali lisainformatsioon, mida soovid sünkroniseerida"
-"For example \"demo\" from aadress https://demo.sendsmaily.net/","Näiteks \"demo\" aadressist https://demo.sendsmaily.net/"
-"API key","Arendusliidese võti (API key)"
-"Sendsmaily subdomain","Sendsmaily alamdomeen"
+"For example &quot;demo&quot; from https://<strong>demo</strong>.sendsmaily.net/","Näiteks &quot;demo&quot; aadressist https://<strong>demo</strong>.sendsmaily.net/"
```

Comment:
There are also files for en_GB and en_US are those necessary? Seems that they are the same translations as used in regular displayed text.

Reply:
I am not sure. Maybe Magento requires them for some reason. What happens if you remove them?

PS. British and US English have different spelling for certain words (like colour and color).

Reply:
Removed both English translations. They didn't seem to change anything. 608fbd51c1b00c0b7b2dfcd11dda1b068a5c5e16